website/content/docs/agent-and-proxy/proxy/caching/static-secret-caching.mdx

```diff
@@ -8,19 +8,25 @@ description: |-
 
 # Vault Proxy static secret caching
 
+@include 'alerts/enterprise-only.mdx'
+
 Use static secret caching with Vault Proxy to cache KVv1 and KVv2 secrets to
 minimize requests made to Vault and provide resilient connections for clients.
 
+Vault Proxy utilizes the Enterprise only [Vault event notification system](/vault/docs/concepts/events)
+feature for cache freshness. As a result, static secret caching can only be used
+with Vault Enterprise installations.
+
 ## Step 1: Subscribe Vault Proxy to KV events
 
 Vault Proxy uses Vault events and auto-auth to monitor secret status and make
 appropriate cache updates.
 1. Enable [auto-auth](/vault/docs/agent-and-proxy/autoauth).
 1. Create an auto-auth token with permission to subscribe to KV event updates
-with the [Vault event system](/vault/docs/concepts/events). For example, to
-create a policy that grants access to static secret (KVv1 and KVv2) events,
-we need permission to subscribe to the `events` endpoint, as well as the
-`list` and `subscribe` permissions on KV secrets we want to get secrets
+with the [Vault event notification system](/vault/docs/concepts/events). For
+example, to create a policy that grants access to static secret (KVv1 and KVv2)
+events, we need permission to subscribe to the `events` endpoint, as well as
```

Comment:
```suggestion
events, you need permission to subscribe to the `events` endpoint, as well as
```
Style correction: reserve "we" for cases where people at HashiCorp are doing things.

Reply:
Thank you! Good catch.